manifests/charts/controlplane/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: controlplane
+description: Helm umbrella chart for Istio controlplane components
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+kubeVersion: ">= 1.23.0-0"
+keywords:
+  - istiod
+  - istio
+  - controlplane
+  - mesh
+sources:
+  - https://github.com/istio/istio
+icon: https://istio.io/latest/favicons/android-192x192.png
+
+# These will be stamped out to real versions during release publish
+dependencies:
+  # TODO it makes the most logical sense for `base`
+  # (which includes CP CRDs, mostly) to be installed
+  # with the controlplane. But, as it is a singleton chart,
+  # it cannot be, without some work.
+  # - name: base
+  #   version: 1.0.0
+  #   repository: "file://../base"
+  - name: istiod
```

Comment:
Since moving `base` means this is now just one chart - it might be better to drop this and just have the "dataplane" chart.

Or something like

- `helm install istio-discovery`
- `helm install ambient-dataplane`

I think 

- `helm install istio-controlplane`
- `helm install istio-dataplane` 

is a bit more obvious and logical, but it's the same chartcount either way.